### ISSUE TITLE
Could not find a part of the path of 'Microsoft.iOS.Sdk'

### ISSUE THREAD

**Issue by maonaoda** [JaneySprings/DotNet.Meteor#118]
```
12:06:43.6348|[Handled] Could not find a part of the path '/usr/local/share/dotnet/packs/Microsoft.iOS.Sdk'.: System.IO.DirectoryNotFoundException: Could not find a part of the path '/usr/local/share/dotnet/packs/Microsoft.iOS.Sdk'.
   at System.IO.Enumeration.FileSystemEnumerator`1.CreateDirectoryHandle(String path, Boolean ignoreNotFound)
   at System.IO.Enumeration.FileSystemEnumerator`1.Init()
   at System.IO.Enumeration.FileSystemEnumerable`1..ctor(String directory, FindTransform transform, EnumerationOptions options, Boolean isNormalized)
   at System.IO.Enumeration.FileSystemEnumerableFactory.UserDirectories(String directory, String expression, EnumerationOptions options)
   at System.IO.Directory.InternalEnumeratePaths(String path, String searchPattern, SearchTarget searchTarget, EnumerationOptions options)
   at System.IO.Directory.GetDirectories(String path, String searchPattern, EnumerationOptions enumerationOptions)
   at DotNet.Meteor.Common.AppleSdk.MLaunchTool() in /Users/runner/work/1/s/src/DotNet.Meteor.Common/Sdk/AppleSdk.cs:line 43
   at DotNet.Meteor.Debug.Sdk.MonoLaunch.DebugSim(String serial, String bundlePath, Int32 port, IProcessLogger logger) in /Users/runner/work/1/s/src/DotNet.Meteor.Debug/Sdk/MonoLaunch.cs:line 72
   at DotNet.Meteor.Debug.DebugLaunchAgent.LaunchAppleMobile(IProcessLogger logger) in /Users/runner/work/1/s/src/DotNet.Meteor.Debug/Agents/DebugLaunchAgent.cs:line 52
   at DotNet.Meteor.Debug.DebugLaunchAgent.Launch(IProcessLogger logger) in /Users/runner/work/1/s/src/DotNet.Meteor.Debug/Agents/DebugLaunchAgent.cs:line 31
   at DotNet.Meteor.Debug.DebugSession.<>c__DisplayClass7_0.<HandleLaunchRequest>b__0() in /Users/runner/work/1/s/src/DotNet.Meteor.Debug/DebugSession.cs:line 75
   at DotNet.Meteor.Debug.Extensions.ServerExtensions.DoSafe[T](Func`1 handler, Action finalizer) in /Users/runner/work/1/s/src/DotNet.Meteor.Debug/Extensions/ServerExtensions.cs:line 101
12:06:43.6389|--> R (launch-2): {"type":"response","request_seq":2,"success":false,"command":"launch","message":"Could not find a part of the path '/usr/local/share/dotnet/packs/Microsoft.iOS.Sdk'.","body":{"error":{"id":0,"format":"Could not find a part of the path '/usr/local/share/dotnet/packs/Microsoft.iOS.Sdk'.","url":"file:///Users/wangyk/.vscode/extensions/nromanov.dotnet-meteor-5.3.1-darwin-arm64/extension/bin/Debug/logs/Debug.log"}},"seq":2}
12:06:43.6415|<--   C (disconnect-3): {"command":"disconnect","arguments":{"restart":false},"type":"request","seq":3}
12:06:43.6440|Disposing <.ctor>b__16_0
12:06:43.6440|--> R (disconnect-3): {"type":"response","request_seq":3,"success":true,"command":"disconnect","body":{},"seq":3}
```

https://github.com/JaneySprings/DotNet.Meteor/blob/2ca0190ff96187959854a348cd75eb729c9b6bb3/src/DotNet.Meteor.Common/Sdk/AppleSdk.cs#L44
---

I'm not sure if it's my environment problem. After I use dotnet-sdk-9.0.100-preview.7.24407.12-osx-arm64, I can't debug anymore.

![image](https://github.com/user-attachments/assets/4e27279a-2bfd-449c-86e9-f8cf82e89f0e)

**Comment by maonaoda** [comment]
After copying and renaming the **Microsoft.iOS.Sdk.net9.0_17.5** to **Microsoft.iOS.Sdk**,
i can debug yet.

**Comment by JaneySprings** [comment]
Hi, @maonaoda ! Yeah, it's a bug

**Comment by maonaoda** [comment]
@JaneySprings 
Yes, I think this needs to be judged comprehensively based on the dotnet version used by the current project.

like using global.json
```
{
  "sdk": {
    "version": "8.0.302"
  }
}
```

**Comment by JaneySprings** [comment]
Fixed in `5.3.2`. Now you can see `mlaunch` version in the debug console (first message):

<img width="722" alt="image" src="https://github.com/user-attachments/assets/2a406ce2-c013-4bac-b8f2-605349620824">

**Comment by JaneySprings** [comment]
Sorry, but VS Marketplace is temporary unavailable. 
https://github.com/microsoft/vscode/issues/226056

I will publish this fix as soon as posible!
